web/modules/custom/dpl_opening_hours/src/Plugin/rest/resource/v1/OpeningHoursLegacyResource.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Drupal\dpl_opening_hours\Plugin\rest\resource\v1;
+
+use DanskernesDigitaleBibliotek\CMS\Api\Model\DplOpeningHoursLegacyListGET200ResponseInner as OpeningHoursLegacyResponse;
+use Drupal\Component\Utility\NestedArray;
+use Drupal\Core\Cache\CacheableResponse;
+use Drupal\dpl_opening_hours\Model\OpeningHoursInstance;
+use Drupal\drupal_typed\RequestTyped;
+use JMS\Serializer\ContextFactory\DefaultSerializationContextFactory;
+use Safe\DateTime;
+use Symfony\Component\DependencyInjection\ContainerInterface;
+use Symfony\Component\HttpFoundation\Request;
+use Symfony\Component\HttpFoundation\Response;
+use Symfony\Component\HttpKernel\Exception\BadRequestHttpException;
+
+/**
+ * Retrieve opening hours.
+ *
+ * @RestResource (
+ *   id = "dpl_opening_hours_legacy_list",
+ *   label = @Translation("List all opening hours for legacy API"),
+ *   uri_paths = {
+ *     "canonical" = "/opening_hours/instances",
+ *   },
+ * )
+ */
+final class OpeningHoursLegacyResource extends OpeningHoursResourceBase {
+
+  /**
+   * The cache tag to use for list operations.
+   */
+  const CACHE_TAG_LIST = 'dpl_opening_hours:list';
```

Comment:
Please remove the cache tag declaration here.

As I see it, the constant replicates what is already available from the base class.